tests/test_utils.py

```diff
@@ -1,5 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
-
```

Comment:
This is very nit but I'd prefer we kept this line for readability (if CI demanded this change, feel free to disregard)

Reply:
`make fix` did this

Reply:
It did not actually some sort of other autoformatting did. I'll restore it, it would have failed CI before if it were a problem.